test/ApplicationInsightsJavaScriptSnippetTest/Rendered.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page - ApplicationInsightsJavaScriptSnippetSample</title>
+    <script>document.write('Head tag script. Look for Application Insights JavaScriptSnippet below this script in the HTML file.')</script>
+    <script type="text/javascript">
```

Comment:
We try to change the snippet script as infrequently as possible but it has happened a couple times.